packages/aws-cdk-lib/aws-efs/lib/efs-file-system.ts

```diff
@@ -350,6 +357,51 @@ export interface FileSystemAttributes {
   readonly fileSystemArn?: string;
 }
 
+/**
+ * Replication configuration for the file system.
+ */
+export interface ReplicationConfiguration {
+  /**
+   * Whether to enable automatic replication.
+   *
+   * Other replication settings(`destinationFileSystem`, `kmsKey`, `region`, `az`) cannot be set if this is set to false.
+   */
+  readonly enable: boolean;
```

Comment:
Is this necessary? 🤔
We don't have a corresponding flag in CFN and it's probably gonna work the same by leaving `replicationConfiguration: undefined`

Reply:
You're right. What am I even doing...

One concern is that configuring replicationConfiguration as `[{}]` can still result in successful replication creation.
```ts
new efs.FileSystem(this, 'ReplicationSourceFileSystem1', {
  vpc,
  replicationConfiguration: [{}] // This means specify region as stack's one
});
```

This won't be a problem, but it does look somewhat odd.
What are your thoughts on this?

Reply:
I think we should leave `readonly replicationConfiguration?: ReplicationConfiguration` for now since we can only define one configuration atm.
It would be cumbersome to define an array for which you can only pass one element.
Btw, can you provide some documentation on `replicationConfiguration` accepting only one configuration?

Reply:
I've reverted the changes to accept `replicationConfiguration` as an object not an array.
And, I've added some notes to README.md. 

Could you please check it?